homeassistant/helpers/device_registry.py

```diff
@@ -33,6 +33,26 @@
 CONNECTION_ZIGBEE = "zigbee"
 
 
+@attr.s(slots=True, frozen=True)
+class DeletedDeviceEntry:
+    """Deleted Device Registry Entry."""
+
+    config_entries: Set[str] = attr.ib()
+    connections: List[Tuple[str, str]] = attr.ib()
+    identifiers: List[Tuple[str, str]] = attr.ib()
+    id: str = attr.ib()
+
+    def to_device_entry(self):
+        """Create DeviceEntry from DeletedDeviceEntry."""
+        return DeviceEntry(
+            config_entries=self.config_entries,
+            connections=self.connections,
+            identifiers=self.identifiers,
```

Comment:
This is not correct. `DeletedDeviceEntry` has them as a `List`, `DeviceEntry` has them as a `Set`. Let's just have `DeletedDeviceEntry` also use sets.

Reply:
Oops! Fixed.